packages/stacked_gql/README.md

```diff
@@ -0,0 +1,14 @@
+# stacked_gql
+
+A new Flutter package project.
+
+## Getting Started
+
+This project is a starting point for a Dart
+[package](https://flutter.dev/developing-packages/),
+a library module containing code that can be shared easily across
+multiple Flutter or Dart projects.
+
+For help getting started with Flutter, view our 
+[online documentation](https://flutter.dev/docs), which offers tutorials, 
+samples, guidance on mobile development, and a full API reference.
```

Comment:
Remove this and change the "A new flutter project" to a package that improves the usability of gql in flutter applications.